pages/index.js

```diff
@@ -26,12 +26,31 @@ const Game = () => {
         winner,
       })
     );
-    
-    // Clone fillBoard constant to avoid any mutation
-    setBoard(board);
+  };
+
+  const newGame = () => {
+    setBoard([
+      Array(7).fill(null),
+      Array(7).fill(null),
+      Array(7).fill(null),
+      Array(7).fill(null),
+      Array(7).fill(null),
+      Array(7).fill(null),
+      Array(7).fill(null),
+    ])
```

Comment:
I don't know why my commit hasn't pulled through but this should be `setBoard([...fillBoard])`